Source/WebCore/platform/graphics/texmap/coordinated/GraphicsLayerCoordinated.h

```diff
@@ -95,6 +95,10 @@ class GraphicsLayerCoordinated final : public GraphicsLayer {
 
     void deviceOrPageScaleFactorChanged() override;
 
+    float rootRelativeScaleFactor() { return m_rootRelativeScaleFactor; }
```

Comment:
```suggestion
    float rootRelativeScaleFactor() const { return m_rootRelativeScaleFactor; }
```

Reply:
Who needs this accessor, btw?

Reply:
A `GraphicsLayerCoordinated` uses its parent's `rootRelativeScaleFactor` to calculate its own:
https://github.com/WebKit/WebKit/pull/39495/files#diff-fd2a9a91ff7f0794e15c1cee3fd7a60b32984fe5a7334bbf02e5efbca2f4d4f0R497